tensorflow/lite/micro/kernels/add_test.cc

```diff
@@ -362,4 +362,132 @@ TF_LITE_MICRO_TEST(QuantizedAddWithMixedBroadcastInt8) {
   }
 }
 
+TF_LITE_MICRO_TEST(QuantizedAddNoActivationInt16) {
+  const float scales[] = {0.25, 0.5, 1.0};
+  const int zero_points[] = {0, 0, 0};
+  const int output_dims_count = 4;
```

Comment:
constexpr, and move closer to first use?

Reply:
And this (mis)use of a const instead of constexpr for array sizes seems to be common across this entire file. Maybe fix all the occurrences in a follow-up PR?

Reply:
I find-and-replaced all "const int output_dims_count" with "constexpr int output_dims_count" here since it's pretty straightforward.